examples/resnet_distributed_torch.yaml

```diff
@@ -9,7 +9,7 @@ num_nodes: 2
 setup: |
     pip3 install --upgrade pip
     git clone https://github.com/michaelzhiluo/pytorch-distributed-resnet
-    cd pytorch-distributed-resnet && pip3 install -r requirements.txt
+    cd pytorch-distributed-resnet && pip3 install -r requirements.txt torch==1.12.1+cu113 --extra-index-url https://download.pytorch.org/whl/cu113
```

Comment:
Q: why pin to 11.3, but not 11.6?

Q: does a 11.6/11.5 image support a torch cu113 wheel?

Reply:
1. We have to pin to 11.3, since the Azure image only supports 11.5 and below, though other images support 11.6
2. Yes, the cuda is backward compatible.